metadata/ComputerVision__5__metadata.json

```diff
@@ -20,7 +20,7 @@
   "minAIFabricVersion": "22.10",
   "languageVersion": 3,
   "version": 5,
-  "customVersion": "23.04.1",
-  "imagePath": "cv2304:23.4.5.6",
+  "customVersion": "23.04.2",
```

Comment:
The first version number should have been `23.04.0` but accidentally I used `.1` instead so now we're stuck for this particular AS release with version numbers 1 higher than the encapsulating AS version number...